privacyidea/lib/policy.py

```diff
@@ -720,7 +720,7 @@ def match_policies(self, name=None, scope=None, realm=None, active=None,
         """
         if user_object is not None:
             # if a user_object is passed, we check, if it differs from potentially passed user, resolver, realm:
-            if (user and user.lower() != user_object.login.lower()) \
+            if (user and user.lower() not in {user_object.login.lower(), user_object.used_login.lower()}) \
```

Comment:
We have not used curly braces for sets anywhere else. This was a bit puzzling to me.
Is there a reason you chose curly braces instead of normal braces?

Reply:
The idea for the `set` was, that we only have one element to compare if the `login` and `used_login` attributes are equal.